docker/cpp-sysroot.dockerfile

```diff
@@ -49,8 +50,14 @@ RUN cd /code/cpp \
         libfaasmpi --native --shared \
     # Install toolchain files
     && inv install \
-    # Build ported third-pary WASM libraries (libc first as it is needed in the
-    # others)
+    # Build Faasm WASM libraries
```

Comment:
I moved the WASM libraries' build before the ported-libraries/apps one, as the latter may depend on the former. E.g. LAMMPS links with `libfaasmpi`.